core/domain/feedback_jobs_continuous.py

```diff
@@ -135,26 +149,25 @@ def get_thread_analytics_multi(cls, exploration_ids):
                 exploration_ids))
 
         return [feedback_domain.FeedbackAnalytics(
-            exploration_ids[i],
-            (realtime_models[i].num_open_threads
-             if realtime_models[i] is not None else 0) +
+            exploration_ids[i], (realtime_models[i].num_open_threads
```

Comment:
That's odd. OK, let's keep the old styling and see if it happens again -- if so, I might be able to advise on the specifics.

Btw, as a tip, you can use backticks in github comments to display things in code font, like this: `hello!` More info [here](https://guides.github.com/features/mastering-markdown/).